receiptui/src/androidTest/java/com/hyperwallet/android/ui/receipt/ListPrepaidCardReceiptsTest.java

```diff
@@ -83,12 +84,14 @@ protected Intent getActivityIntent() {
 
     @Before
     public void setup() {
-        Hyperwallet.getInstance(new TestAuthenticationProvider());
-
         mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
                 .getResourceContent("authentication_token_response.json")).mock();
 
         setLocale(Locale.US);
+    }
+
+    @Before
+    public void registerIdlingResource() {
```

Comment:
I think there's a method already annotated with @Before. Could we add the idling registration there instead? 
If yes, then the same needs to be done to other tests

Reply:
Updated